wiki/Beatmap_ranking_procedure/de.md

```diff
@@ -0,0 +1,60 @@
+---
+tags:
+  - qualification
+  - nomination
+  - nominations
+  - nom
+  - ranking
+  - ranked
+  - Qualifizierung
+  - Nominierung
+  - gerankt
+---
+
+# Ranking-Verfahren für Beatmaps
+
+*Siehe auch: [Rank (Begriffsabgrenzung)](/wiki/Disambiguation/Rank)*
+
+Bevor [Beatmaps](/wiki/Beatmap) in die Kategorie [Ranked](/wiki/Beatmap/Category#ranked) eingestuft werden können, müssen sie von der Community überprüft werden, indem sie das Ranking-Verfahren durchlaufen.
+
+## Feedback
+
+Der Ersteller einer Beatmap kann seine Einsendungen als `Work in Progress` oder `Ausstehend` kennzeichnen. Die osu!-Community soll Feedback zu Beatmaps geben, die den beiden Kategorien zugeordnet sind.
+
+Durch [Modding](/wiki/Modding) erhält man konstruktive Kritik für eine Beatmap und kann so die Qualität der Beatmap steigern. Üblicherweise veröffentlichen Nutzer ihr Feedback auf der [Diskussionsseite](/wiki/Beatmap_discussion) einer Beatmap oder reden mit dem Ersteller direkt über Probleme.
+
+Die Qualitätsstandards für die Ranked-Kategorie sind hoch, beim Modding müssen also viele Punkte beachtet werden. Manchmal kann es auch passieren, dass Beatmaps komplett neu gemacht werden müssen.
+
+Bevor die Beatmap für das Ranking nominiert wird, müssen mindestens 5 verschiedene Nutzer der Beatmap einen [Hype](/wiki/Beatmap/Hype) vergeben.
+
+## Nominierungen {id=nominations}
+
+::: Infobox
+![](img/nomination-progress-beatmapset1857663.png "Der Rankingstatus einer Beatmap auf der Diskussionsseite, wobei der Hype-Train und die Nominierungsleiste zum Teil gefüllt sind.")
```

Comment:
In a quick search, I was not able to find a beatmap ranked status similar to this one (for providing the image in German).

Reply:
I did find [one](https://osu.ppy.sh/beatmapsets/2215702/discussion/-/generalAll/total) as of 2025-16-03.

![grafik](https://github.com/user-attachments/assets/0cdcabea-f6c5-40f9-9ce2-aa5d3655a111)

The localisation is not good here due to the date library used in osu-web doing weird stuff which is not configurable (`Vor vor`). This is not easy to fix (or find a work-around) because these strings are used in so many places, in various different contexts. So changing it in just one place might screw things up in another.

Reply:
This date localization is a known "problem" that the other languages (including english IIRC) also need to work with. It just has to be incorporated into the translations over on Crowdin

Reply:
Regarding the image, maybe keep the English one for now (German one is almost definitely way outdated). I think these show up when your own beatmap is submitted and gets hyped/nominated/qualified, so we might need to ask someone to take a screenshot of this in German.

Reply:
I don't think it's vital here if there's a message below the progress bars (the progress bars are the focus of this image).

Nevertheless, I would not include this exact image due to the grammar issue.